webrtc/src/benchmarkRtc.ts

```diff
@@ -0,0 +1,168 @@
+import { chromium, Browser } from "playwright";
+import { Client } from "./client";
+import { Args } from "./types";
+import { getEncodingsReport, onEncodingsUpdate } from "./encodingReporter";
+
+const frontendAddress = "http://localhost:5005";
+const fakeVideo = "media/sample_video.mjpeg";
+const fakeAudio = "media/sample_audio.wav";
+const ENCODING_REPORT_PERIOD = 5;
+const INBOUD_TRACK_BANDWIDTH = 0.15 + 0.5 + 1.5;
+const OUTBOUND_TRACK_BANDWIDTH = 1.5;
+
+const delay = (s: number) => {
+  return new Promise((resolve) => setTimeout(resolve, 1000 * s));
+};
```

Comment:
What do you think about that?
```suggestion
const second = 1000

const delay = (n: number) => {
  return new Promise((resolve) => setTimeout(resolve, n * second));
};
```